robot-server/robot_server/deck_configuration/store.py

```diff
@@ -26,3 +27,10 @@ async def get(self) -> models.DeckConfigurationResponse:
         return models.DeckConfigurationResponse.construct(
             cutoutFixtures=self._cutoutFixtures, lastUpdatedAt=self._last_updated_at
         )
+
+    def get_deck_configuration(self) -> DeckConfigurationType:
+        """Get the robot's current deck configuration in an expected typing."""
+        deck_configuration: DeckConfigurationType = []
+        for item in self._cutoutFixtures:
+            deck_configuration.append((item.cutoutId, item.cutoutFixtureId))
+        return deck_configuration
```

Comment:
Python style thing: this imperative loop can be simplified into a list comprehension.

```suggestion
        return [(item.cutoutId, item.cutoutFixtureId) for item in self._cutoutFixtures]
```